src/guide/receive_payment.md

```diff
@@ -321,3 +316,30 @@ When waiting for a payment to be received, it is enough to wait for a <code>Paym
 ```
 </section>
 </custom-tabs>
+
+## Event Flows
+Once a receive payment is initiated, you can follow and react to the different payment events using the guide below for each payment method. See [Listening to events](/guide/events.html) for how to subscribe to events.
+
+### Lightning
+| Event | Description | UX Suggestion |
+| --- | --- | --- |
+| **PaymentPending** | The swap service has intercepted a HTLC for the Lightning invoice and broadcast a lockup transaction. The SDK has seen the lockup transaction and will broadcast the claim transaction, either when the lockup transaction is confirmed or if it is accepted as a zero-conf payment. | Show payment as pending. |
+| **PaymentWaitingConfirmation** | The claim transaction has been broadcast or a direct Liquid transaction (<a target="_blank" href="https://docs.boltz.exchange/v/api/magic-routing-hints">MRH</a>) has been seen. | Display successful payment feedback. |
+| **PaymentSucceeded** | The claim transaction or direct Liquid transaction (<a target="_blank" href="https://docs.boltz.exchange/v/api/magic-routing-hints">MRH</a>) is confirmed. | Show payment as complete. |
+| **PaymentFailed** | The swap has failed from one of several reasons. Either the swap/invoice has expired or the lockup transaction failed to broadcast. |  |
+
+### Bitcoin
+| Event | Description | UX Suggestion |
+| --- | --- | --- |
+| **PaymentWaitingFeeAcceptance** | The swap service has seen the Bitcoin lockup transaction. If there was no amount set for this swap or the Bitcoin lockup transaction contains a different amount, the fee has to be accepted, see [Amountless Bitcoin Payments](#amountless-bitcoin-payments). If the fee is within the configured leeway, the swap will be automatically accepted. | Allow the user to review fees for this payment. |
```

Comment:
Currently we are not handling this case for non amount-less swap and I am not sure if we will handle it ATM. So I think we should limit this explanation to amount-less swaps only.